distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
+
+==== Directory Permissions
+
+Restrict access to sensitive files by ensuring that the only users with access privileges are administrators.
+
+===== Directory Permissions on Windows
+
+. Right-click on the file or directory noted below then select *Full Control -> Administrators -> System*.
+. Click *Properties -> Security -> Advanced* and select `Creator Owner` for `${branding}_HOME` (e.g., `C:\ddf`).
```

Comment:
I would love to normalize it, because it's called many things in many different places. I like the neutrality of `<DISTRIBUTION_INSTALL_DIR>`.